src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -941,37 +937,39 @@ rpmostree_compose_builtin_tree (int             argc,
                                cancellable, error))
     goto out;
 
-  {
-    const char *gpgkey;
-    gboolean selinux = TRUE;
-    g_autoptr(GVariant) metadata = NULL;
-
-    g_variant_builder_add (metadata_builder, "{sv}",
-                           "rpmostree.inputhash",
-                           g_variant_new_string (new_inputhash));
+  /* Insert our input hash */
+  g_hash_table_replace (metadata_hash, g_strdup ("rpmostree.inputhash"),
+                        g_variant_ref_sink (g_variant_new_string (new_inputhash)));
 
-    metadata = g_variant_ref_sink (g_variant_builder_end (metadata_builder));
-
-    if (!_rpmostree_jsonutil_object_get_optional_string_member (treefile, "gpg_key", &gpgkey, error))
-      goto out;
+  const char *gpgkey;
```

Comment:
Shouldn't this be initialized? Looking at `_rpmostree_jsonutil_object_get_optional_boolean_member()`, it seems it only assigns a value if it finds the key.